test/shift_test.rb

```diff
@@ -0,0 +1,24 @@
+require "./test/test_helper"
+require "minitest/autorun"
+require "minitest/pride"
+require "mocha/minitest"
+require "./lib/shift"
+
+class ShiftTest < Minitest::Test
+  def test_it_exists
+    shift = Shift.new
+    assert_instance_of Shift, shift
+  end
+
+  def test_it_has_attributes
+    Date.stubs(:today).returns(Date.new(2020, 02, 12))
+    shift = Shift.new
+
+    #IDK how to test for key
```

Comment:
You should ask an instructor about ideas. If you just stubbed key, I don't think that's an appropriate use for stubs. Maybe you can stub rand()?